test_flow.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+# Setup phase - Set the correct user.email and user.name and activate the ssh agent
+eval "$(ssh-agent)" && ssh-add
+git config --global user.email "$GIT_MAIL"
+git config --global user.name "$GIT_NAME"
+
+# Setup phase - Also remember to set your $WANDB_API_KEY (More info in the readme)
+[[ -z "$WANDB_API_KEY" ]] && (echo set \"export WANDB_API_KEY=[your-wandb-key]\" which can be found here: https://wandb.ai/settings; exit;)
```

Comment:
the key is in .netrc it's not alwys necessary to explicitly give it.
is WANDB_API_KEY always set?

Reply:
ok wait. this is all the docker test right?
if so put them hidden in a subfolder.

Reply:
moved